concourse/pipelines/create-deployer.yml

```diff
@@ -280,6 +287,28 @@ jobs:
         params:
           file: generated-concourse-cert/concourse-cert.tar.gz
 
+    - task: generate-concourse-ssh-key
+      config:
+        image: docker:///governmentpaas/curl-ssl
+        inputs:
+        - name: paas-cf
+        outputs:
+        - name: concourse-ssh-key
+        params:
+          AWS_DEFAULT_REGION: {{aws_region}}
+        run:
+          path: sh
+          args:
+          - -e
+          - -c
+          - |
+            apk add --update openssh
+            cd concourse-ssh-key
+            ssh-keygen -t rsa -b 4096 -f generated_concourse_id_rsa -N ''
+            ../paas-cf/concourse/scripts/s3init.sh {{state_bucket}} concourse_id_rsa generated_concourse_id_rsa
+            ../paas-cf/concourse/scripts/s3init.sh {{state_bucket}} concourse_id_rsa.pub generated_concourse_id_rsa.pub
```

Comment:
Can we use `put` to upload the generated key instead of `s3init`?

I know we do it in the other task, but it's a misuse of `s3init` that would be good for us to phase out.